src/main/kotlin/ar/edu/unahur/obj2/vendedores/Centros.kt

```diff
@@ -0,0 +1,27 @@
+package ar.edu.unahur.obj2.vendedores
+
+import java.lang.Exception
+
+class Centro(val ciudad: Ciudad) {
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor: Vendedor) {
+        if (!vendedores.contains(vendedor)) {
+            vendedores.add(vendedor)
+        } else throw Exception("El vendedor ya esta incluido")
+    }
```

Comment:
Una buena practica es poner primero el caso para el que rompa. 

```suggestion
    fun agregarVendedor(vendedor: Vendedor) {
        if (vendedores.contains(vendedor)) throw Exception("El vendedor ya esta incluido")
        vendedores.add(vendedor)
    }
```